### ISSUE TITLE
src/parse/rules/java_rules.build_defs:449:22: error: name 'licences' is not defined

### ISSUE THREAD

**Issue by cameronbraid** [thought-machine/please-examples#8]
Cant run example

./pleasew --version
Please version 13.4.5

```
> ./pleasew  test -i java -o proto.language:java
13:32:50.651 WARNING: Error in config file: warning:
can't store data at section "proto", variable "pythonpackage"

Build stopped after 30ms. 1 target failed:
    //third_party/java:all
src/parse/rules/java_rules.build_defs:449:22: error: name 'licences' is not defined
            filename=filename,
            licences=licences,
                     ^
        )

Traceback:
                                   src/parse/rules/java_rules.build_defs:449:22:               licences=licences,
                                   src/parse/rules/java_rules.build_defs:441:16:           return maven_jar(
                                   src/parse/rules/java_rules.build_defs:403: 5:       if combine:
/Users/cameronbraid/workspace-new/please-examples/third_party/java/BUILD: 40: 1:   maven_jars(
```

**Comment by peterebden** [comment]
Yup, this is broken in the last release but fixed on master - will do another one soon. Apologies!

**Comment by cameronbraid** [comment]
verified fixed in v13.4.6
